source/quickguides/generate_quick_guides.py

```diff
@@ -10,6 +10,14 @@
         'html_path': '/var/www/acme/acme-diags/<username>/',
         'web_address': 'https://acme-viewer.llnl.gov/<username>/',
     },
+    'chrysalis': {
+        'machine_name': 'Chrysalis',
+        'activation_path': '/lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified.sh',
+        'obs_path': '/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags',
+        'test_data_path': '/lcrc/soft/climate/e3sm_diags_data/test_model_data_for_acme_diags/',
+        'html_path': '/lcrc/group/acme/public_html/diagnostic_output/<username>',
```

Comment:
I think this should be `/lcrc/group/e3sm/public_html/diagnostic_output/<username>/`. Note `e3sm` and the ending `/`

Reply:
Thanks for the catch!

Reply:
I pulled this info from `quick-guides-general.rst` for Anvil/blues. Is  `<html_path>` also incorrect here?

`Anvil/blues
^^^^^^^^^^^
<activation_path>: ``source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified.sh``

<obs_path>: ``/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/``

<test_data_path>: ``/lcrc/soft/climate/e3sm_diags_data/test_model_data_for_acme_diags/``

<html_path>: ``/lcrc/group/acme/public_html/diagnostic_output/<username>``

<web_address>: ``https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/<username>``
`

Reply:
The old path with `acme` in it still works, since we are on it, let's use `e3sm` to replace `acme` in `html_path`.

Reply:
This reminds me that in `quick-guide-cori` we will need to change
`#SBATCH --account=acme` to #SBATCH --account=e3sm`. Tom would help make the change in this PR?

Reply:
I can bundle those changes in this PR since they are small. Thanks for clarifying Jill.